roles/container-engine/containerd/defaults/main.yml

```diff
@@ -17,7 +17,7 @@ containerd_package: 'containerd.io'
 containerd_cfg_dir: /etc/containerd
 
 # Path to runc binary
-runc_binary: /usr/sbin/runc
+runc_binary: /usr/bin/runc
```

Comment:
This is probably the wrong place to set the variable. But what would be the right place?

Reply:
OS specific variables are in vars/*.yml, I don't think it's necessary to change the default value for all OS platforms.